components/section/info/src/index.scss

```diff
@@ -0,0 +1,38 @@
+@import '~@schibstedspain/theme-basic/lib/index';
+@import './settings';
+
+.sui-SectionInfo {
+  @include media-breakpoint-down(m) {
+    flex-direction: column;
+  }
+  
+  border-bottom: 1px solid $c-sui-section-info-border;
+  display: flex;
+  flex-direction: row;
+  max-width: 670px;
+  padding: $p-xxl 0;
+
+  &-title {
+    @include media-breakpoint-down(m) {
+      flex-basis: auto;
+      padding: $p-l 0;
+      text-align: center;
+    }
+    
+    flex-basis: 30%;
```

Comment:
This value and the one from line [33](https://github.com/SUI-Components/sui-components/blob/80da9f77393bf9121051974e9435a06cadf141a6/components/section/info/src/index.scss#L33) could be saved as variables as well.